.gitignore

```diff
@@ -20,3 +20,6 @@
 /rpc/src/messages/*.rs
 !/rpc/src/messages/mod.rs
 /rpc/tests/protobuf/
+
+/contracts/*
```

Comment:
Are there build artifacts that are going in here?

Reply:
The purpose of this directory is to allow user created contracts to be edited on the host and available in each container. I don't think there would be a reason for them to be committed, so I excluded it (with exception of the examples dir, where we have contracts that are associated with tests and demos)